mybatis-spring-boot-autoconfigure/src/test/java/org/mybatis/spring/boot/autoconfigure/MybatisAutoConfigurationTest.java

```diff
@@ -106,6 +107,34 @@ public void testWithConfigFile() {
 	}
 
 	@Test
+	public void testWithCheckConfigLocationFileExists() {
+		EnvironmentTestUtils
+				.addEnvironment(this.context, "mybatis.config:mybatis-config.xml",
+						"mybatis.check-config-location=true");
+		this.context.register(EmbeddedDataSourceConfiguration.class,
+				MybatisAutoConfiguration.class);
+		this.context.refresh();
+		assertEquals(1, this.context.getBeanNamesForType(SqlSessionFactory.class).length);
+	}
+
+	@Test
+	public void testWithCheckConfigLocationFileDoesNotExists() {
+		EnvironmentTestUtils.addEnvironment(this.context, "mybatis.config:foo.xml",
+				"mybatis.check-config-location=true");
+		this.context.register(EmbeddedDataSourceConfiguration.class,
+				MybatisAutoConfiguration.class);
+		try {
```

Comment:
@eddumelendez 

I've modified to use the `ExpectedException`.
Please review again.